.sqlfluff

```diff
@@ -0,0 +1,143 @@
+[sqlfluff]
```

Comment:
Question (non-blocking): Are we duplicating the .sqlfluff file here? If so, I'd prefer to have a single source of truth instead of two places to maintain :)

Reply:
Good catch! I removed it from the other repo and also created a `requirements.txt` file here for usability